pkg/kv/dynamodb/store.go

```diff
@@ -195,12 +195,15 @@ func (s *Store) Get(ctx context.Context, partitionKey, key []byte) (*kv.ValueWit
 		return nil, kv.ErrMissingKey
 	}
 
+	start := time.Now()
 	result, err := s.svc.GetItemWithContext(ctx, &dynamodb.GetItemInput{
 		TableName:      aws.String(s.params.TableName),
 		Key:            s.bytesKeyToDynamoKey(partitionKey, key),
 		ConsistentRead: aws.Bool(true),
 	})
+	dynamoRequestDuration.WithLabelValues("GetItem").Observe(time.Since(start).Seconds())
```

Comment:
Why is the consumed capacity not counted for in this flow?

Reply:
Missed it - thanks!